easybuild/easyconfigs/f/FASTA/FASTA-36.3.8i-foss-2022b.eb

```
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = "FASTA"
+version = "36.3.8i"
+
+homepage = 'http://fasta.bioch.virginia.edu'
+description = """The FASTA programs find regions of local or global (new) similarity between 
+protein or DNA sequences, either by searching Protein or DNA databases, or by identifying 
+local duplications within a sequence."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+sources = [{
+    'filename': 'FASTA-36.3.8i.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/wrpearson',
+        'repo_name': 'fasta36',
+        'commit': 'bd53d2af21b21deb20cfb033d4f3e991f5c61a91',
+    },
+}]
```

`git_config` doesn't produce stable checksums, since git doesn't keep track of file timestamps
```suggestion
source_urls = ['https://github.com/wrpearson/fasta36/archive/']
sources = ['v%%(version)s_%s.tar.gz' % local_version_date]
checksums = ['b4b1c3c9be6beebcbaf4215368e159d69255e34c0bdbc84affa10cdb473ce008']
```